modular_skyrat/modules/gun_cargo/code/objects/guns/oldarms.dm

```diff
@@ -0,0 +1,65 @@
+/obj/item/gun/ballistic/automatic/m16/oldarms
+	name = "\improper Mk-11.4 Rifle"
+	desc = "An old-fashioned rifle from Sol-3's bygone era, rumor has it, it can shoot apart an entire jungle, or desert given the time, has \"Keep out of water\" laser-engraved on the side. Now including a free reflex sight!"
```

Comment:
```suggestion
	desc = "An old-fashioned rifle from Sol-3's bygone era. Rumor has it that it can shoot apart an entire jungle (or desert, given the time). It has \"Keep out of water\" laser-engraved on the side. Now including a free reflex sight!"
```